irods/src/main/java/ch/cyberduck/core/irods/IRODSCopyFeature.java

```diff
@@ -43,35 +39,20 @@ public IRODSCopyFeature(final IRODSSession session) {
     }
 
     @Override
-    public Path copy(final Path source, final Path target, final ch.cyberduck.core.transfer.TransferStatus status, final ConnectionCallback callback, final StreamListener listener) throws BackgroundException {
+    public Path copy(final Path source, final Path target, final TransferStatus status,
+                     final ConnectionCallback callback, final StreamListener listener) throws BackgroundException {
         try {
-            final IRODSFileSystemAO fs = session.getClient();
-            final DataTransferOperations transfer = fs.getIRODSAccessObjectFactory()
-                .getDataTransferOperations(fs.getIRODSAccount());
-            transfer.copy(fs.getIRODSFileFactory().instanceIRODSFile(source.getAbsolute()),
-                fs.getIRODSFileFactory().instanceIRODSFile(target.getAbsolute()), new TransferStatusCallbackListener() {
-                    @Override
-                    public FileStatusCallbackResponse statusCallback(final TransferStatus transferStatus) {
-                        return FileStatusCallbackResponse.CONTINUE;
-                    }
+            final IRODSConnection conn = session.getClient();
+            final String from = source.getAbsolute();
+            final String to = target.getAbsolute();
 
-                    @Override
-                    public void overallStatusCallback(final TransferStatus transferStatus) {
-                        switch(transferStatus.getTransferState()) {
-                            case OVERALL_COMPLETION:
-                                listener.sent(status.getLength());
-                        }
-                    }
+            // TODO If we're dealing with a collection, should existing data objects sharing
```

Comment:
You should assume override if `status.isExists()` and fail otherwise for existing target files.